src/App.js

```diff
@@ -1,25 +1,61 @@
-import logo from './logo.svg';
-import './App.css';
-
+import React, { useEffect } from 'react';
+import { Alert, Fade } from 'react-bootstrap';
+import { BrowserRouter, Switch } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { useCookies } from 'react-cookie';
+import Dashboard from './components/Dashboard/Dashboard';
+import HomeScreen from './components/HomeScreen/HomeScreen';
+import Search from './components/Search/Search';
+import Settings from './components/Settings/Settings';
+import PrivateRoute from './components/PrivateRoute/PrivateRoute';
+import PublicRoute from './components/PublicRoute/PublicRoute';
```

Comment:
please use the absolute paths for imports

Reply:
fixed